backup_and_restore/application_backup_and_restore/oadp-api.adoc

```diff
@@ -0,0 +1,258 @@
+:_content-type: ASSEMBLY
+[id="oadp-api"]
+= APIs used with OADP
+include::_attributes/common-attributes.adoc[]
+:context: oadp-api
+:oadp-api:
+:namespace: openshift-adp
+:local-product: OADP
+:velero-domain: velero.io
+
+toc::[]
+
+The document provides information about the following APIs that you can use with OADP:
+
+* Velero API
+* OADP API
+
+== Velero API
```

Comment:
These headings need [anchors](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-in-assembly-files).